### ISSUE TITLE
columns for paragraphs

### ISSUE THREAD

**Issue by pecirep** [dolanmiu/docx#423]
currently columns can only be set for sections, which only one setting is possible per page. Would it be possible to allow column settings for paragraphs, so there can for example be a paragraph with 1 column and a paragraph with 2 columns on the same page?

**Comment by dolanmiu** [comment]
Sort of,

try this:

https://github.com/dolanmiu/docx/blob/master/demo/44-multiple-columns.ts

**Comment by pecirep** [comment]
That demo shows the column property for sections, but I cannot have 2 sections on the same page (right?)
Is it possible to add the column property to the paragraph object so I can have a paragraph with only one column and a paragraph with 2 columns in a single section?

**Comment by dolanmiu** [comment]
Not possible just yet im afraid

**Comment by deathg0d** [comment]
+1 for this feature

**Comment by dolanmiu** [comment]
It is now possible to have two sections on the same with this PR:

https://github.com/dolanmiu/docx/pull/780

Will be released in `6.0.0`!

**Comment by mwashambw** [comment]
> It is now possible to have two sections on the same with this PR:
> 
> #780
> 
> This should address this issue.
> 
> Will be released in `6.0.0`!

Hi,
Is there any example on this feature? (adding the column property to the paragraph object so I can have a paragraph with only one column and a paragraph with 2 columns in a single section)

Because I don't see column property in paragraph object

**Comment by anti-the-social** [comment]
I don't remember exactly, but I think columns is the property only for sections.
But the answer for paragraph column is section with `type: SectionType.CONTINUOUS` property.
So this way you have sections going one after another one.

```
doc.addSection({
    properties: {
        type: SectionType.CONTINUOUS,
    }
    children: [
        new Paragraph({
            children: [new TextRun("Hello World")],
        }),
    ],
});
```

Reference: https://github.com/dolanmiu/docx/blob/858a994d134d1bd81c4881d5fc48a0b9977a0d2f/docs/usage/sections.md
